src/stylesheets/tk-form-fields.scss

```diff
@@ -6,20 +6,14 @@
 @import 'bootstrap-sass/assets/stylesheets/bootstrap/forms';
 @import 'bootstrap-sass/assets/stylesheets/bootstrap/buttons';
 
-%tk-form-fields {
-  @import "variables";
-  // Form fields partials
-  @import "tkff-fieldsets";
-  @import "tkff-labels";
-  @import "tkff-inputs";
-  @import "tkff-general-validations";
-  @import "tkff-buttons";
-  @import "tkff-textareas";
+@import 'variables';
+@import 'placeholders';
+@import 'themes/generator';
 
-  @include tkff-buttons;
-  @include tkff-fieldsets;
-  @include tkff-requirements;
-  @include tkff-labels;
-  @include tkff-textarea;
-  @include tkff-inputs;
+%tk-form-fields--default {
+  @include tk-fields-theme($gray-2);
+}
+
+%tk-form-fields--light {
```

Comment:
If this theme just changes the bg to white, maybe it should be `--white` ? Since the default gray bg is still fairly light.